app/src/components/googleMaps/DirectionButton.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+import ArrowCircleRight from "../../assets/arrow-circle-right.svg";
+import MapModal from "./MapModal";
+
+
+// To use the button from an events detail page
+// Write <DirectionButton id={int}/>
+// The int refers to the concert id in the concert_details db view
+
+export default function Directions(props) {
+    const id = parseInt(props['id']);
+    const [show, setShow] = useState(false);
```

Comment:
show sounds like it has to do with music show, since its a music app, maybe isVisible and setIsVisible works better if following JS conventions?